### DISCUSSION TITLE
Edge Context Menu disappeared

### DISCUSSION THREAD

**Discussion by jnoyd** [RoelVB/ChromeKeePass#111] [General]
Hello. Noticed today that on Edge the context menu (right click) options for EdgeKeePass have disappeared. Not sure if this was caused by an edge update or some other outside force (IT guys enforcing Edge settings maybe <shrug>). Anyway, managed to fix it while typing this up but since I'm this far I figure I might as well leave a breadcrumb on what fixed it for me. I removed the extension, re-installed the extension, and relinked it to keepass. The context menu option is back. Hope that helps someone.

**Comment by RoelVB** [comment]
Strange. Thank you for sharing
